data/hai/hai reveal 2 start.txt

```diff
@@ -10,10 +10,34 @@
 
 ########################################################################
 
-#Investigate a theorised smuggling route and rescue Hai. You have to do at least three of these missions.
+#Investigate a theorized smuggling route and rescue Hai. You have to do at least three of these missions.
 
 #NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.
 
+mission "Hai Rescue: Yeertle Hint"
+	landing
+	name "Investigate a Were-Squirrel"
+	description "A truly outlandish rumor has been floating around that might be a Hai."
+	source
+		near "Betelgeuse" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+	destination Prime
+	to offer
+		"combat rating" > 50
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Yeertle Family: offered"
+		not "event: hai-human treaty signed"
+		random < 50
+	on offer
+		conversation
+			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. Often this contains useful tidbits like, which merchants are ticked off at the moment, and whether you should expect to wait a while for refueling. Today though you overhear one captain, obviously having encountered a common acquaintance relaying a very remarkable tale.`
```

Comment:
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Often this contains useful tidbits, like which merchants are ticked off at the moment and whether you should expect to wait a while for refueling. Today, though, you overhear one captain, obviously having encountered a common acquaintance relaying a very remarkable tale.`
```
OR
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Often this contains useful tidbits, such as which merchants are ticked off at the moment and whether you should expect to wait a while for refueling. Today, though, you overhear a captain telling an acquaintance a remarkable tale.`
```
As per norm, top is purely corrections, bottom is my suggestions for flow.